catboost/python-package/ut/medium/test.py

```diff
@@ -2903,6 +2903,8 @@ def test_set_cat_features_in_init():
     label = np.random.randint(2, size=20)
     train_pool = Pool(data, label, cat_features=[1, 2])
     test_pool = Pool(data, label, cat_features=[1, 2])
+    #for single tests
```

Comment:
Please revert changes in this test, and add the tests described in the previous comment.